dired-sidebar.el

```diff
@@ -800,14 +800,8 @@ the relevant file-directory clicked on by the mouse."
     ;; Use `project' if `projectile' is not loaded yet.
     ;; `projectile' is a big package and takes a while to load so it's better
     ;; to defer loading it as long as possible (until the user chooses).
-    (dired-sidebar-if-let* ((project (project-current)))
-        ;; https://github.com/jojojames/dired-sidebar/issues/61
-        (if (eq (type-of project) 'ede-proj-project)
```

Comment:
I test `project-root` works on `ede-proj-project` with steps in https://github.com/jojojames/dired-sidebar/issues/61#issuecomment-835749193